libs/types/src/content-announcement/types.gen.ts

```diff
@@ -7,7 +7,6 @@ export enum AnnouncementType {
   Reaction = 4,
   Profile = 5,
   Update = 6,
-  PublicFollows = 113,
```

Comment:
Has this moved to a different announcement type? Flagging because SAT uses public follows.

Reply:
this is auto generated. I'm not sure. @JoeCap08055 do you have any idea?